docs/user-guide/prerequisite.md

```diff
@@ -0,0 +1,27 @@
+###...
+
+# Prerequisites
+## VMWare Environment - vSphere/Photon
```

Comment:
Let's remove references to Photon everywhere. Let's add a separate section for Photon support.